docs/concepts/index.md

```
@@ -21,9 +21,7 @@ Each assessment contains:
 
 ### Concept Map
 
-<div style="max-width: 500px;" markdown>
-![](./concept-map-example.d2){pad="0"}
-</div>
+![](./concept-map-example.d2){pad="0" scale="1"}
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)